### DISCUSSION TITLE
Item in Anvil

### DISCUSSION THREAD

**Discussion by MrKakotu** [Zrips/CMI#4666] [Q&A]
**Question: **
So, i wonder if there is any way to disable colored name in anvil. I have set **cmi.anvil.colors** to false that works.
But if i have a pickaxe named "Pickaxe" in the color &4. If i add mending or any other enchant to it, it will still stay the same itemname. What i want is that if i add an enchantment to the pickaxe, i want the name to change to "&4Pickaxe" without color.
Is this possible or may it be a feature later?
![2020-11-29_15 40 54](https://user-images.githubusercontent.com/65896836/100544909-4f5e7280-3259-11eb-9c4d-696a8a660b84.png)
![2020-11-29_15 40 28](https://user-images.githubusercontent.com/65896836/100544910-508f9f80-3259-11eb-94a3-edda298fa041.png)
![2020-11-29_15 40 30](https://user-images.githubusercontent.com/65896836/100544912-51283600-3259-11eb-829c-45c4b9eb0276.png)
![2020-11-29_15 40 41](https://user-images.githubusercontent.com/65896836/100544913-52596300-3259-11eb-9429-7d5859660ee2.png)
![2020-11-29_15 40 48](https://user-images.githubusercontent.com/65896836/100544914-52f1f980-3259-11eb-8f49-efd6bcbde1c3.png)

**Comment by mercurialmusic** [comment]
Keeping the name is normal vanilla behavior. If added this would definitely need to be a config option since it would be quite annoying in most cases and I'd rather not have to randomly add a permission node for players to be able to keep the existing name format for their tool.

**Comment by mrfloris** [comment]
<img width="1086" alt="Screenshot 2021-09-08 at 10 13 39" src="https://user-images.githubusercontent.com/28841349/132472036-532a3e9b-4fdf-498e-b519-e98da0c57d50.png">

These are the perms related to anvil
